docsite/rst/playbooks_filters.rst

```diff
@@ -316,28 +338,32 @@ To get the directory from a path::
 
     {{ path | dirname }}
 
-To expand a path containing a tilde (`~`) character (new in version 1.5)::
+To expand a path containing a tilde (`~`) character (since 1.5)::
```

Comment:
we use 'new in version' in most other pages, I think we should keep it consistent